src/main/app/claims/ccjModelConverter.ts

```diff
@@ -0,0 +1,56 @@
+import { DraftCCJ } from 'ccj/draft/DraftCCJ'
```

Comment:
It would be good if all our `*.ts` files started with a lower case letter. Having mixed case leads to hard to debug problems with macOS being case sensitive and Linux (Jenkins) being case sensitive.

Reply:
I agree. All files I added start with lower case. 

I'd change it in a separate PR.

Reply:
👍